dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -9833,4979 +9833,4982 @@ BEGIN
-    43951 "Cherokee Small Letter Tli"
-    43952 "Cherokee Small Letter Tlo"
-    43953 "Cherokee Small Letter Tlu"
-    43954 "Cherokee Small Letter Tlv"
-    43955 "Cherokee Small Letter Tsa"
-    43956 "Cherokee Small Letter Tse"
-    43957 "Cherokee Small Letter Tsi"
-    43958 "Cherokee Small Letter Tso"
-    43959 "Cherokee Small Letter Tsu"
-    43960 "Cherokee Small Letter Tsv"
-    43961 "Cherokee Small Letter Wa"
-    43962 "Cherokee Small Letter We"
-    43963 "Cherokee Small Letter Wi"
-    43964 "Cherokee Small Letter Wo"
-    43965 "Cherokee Small Letter Wu"
-    43966 "Cherokee Small Letter Wv"
-    43967 "Cherokee Small Letter Ya"
-    43968 "Meetei Mayek Letter Kok"
-    43969 "Meetei Mayek Letter Sam"
-    43970 "Meetei Mayek Letter Lai"
-    43971 "Meetei Mayek Letter Mit"
-    43972 "Meetei Mayek Letter Pa"
-    43973 "Meetei Mayek Letter Na"
-    43974 "Meetei Mayek Letter Chil"
-    43975 "Meetei Mayek Letter Til"
-    43976 "Meetei Mayek Letter Khou"
-    43977 "Meetei Mayek Letter Ngou"
-    43978 "Meetei Mayek Letter Thou"
-    43979 "Meetei Mayek Letter Wai"
-    43980 "Meetei Mayek Letter Yang"
-    43981 "Meetei Mayek Letter Huk"
-    43982 "Meetei Mayek Letter Un"
-    43983 "Meetei Mayek Letter I"
-    43984 "Meetei Mayek Letter Pham"
-    43985 "Meetei Mayek Letter Atiya"
-    43986 "Meetei Mayek Letter Gok"
-    43987 "Meetei Mayek Letter Jham"
-    43988 "Meetei Mayek Letter Rai"
-    43989 "Meetei Mayek Letter Ba"
-    43990 "Meetei Mayek Letter Jil"
-    43991 "Meetei Mayek Letter Dil"
-    43992 "Meetei Mayek Letter Ghou"
-    43993 "Meetei Mayek Letter Dhou"
-    43994 "Meetei Mayek Letter Bham"
-    43995 "Meetei Mayek Letter Kok Lonsum"
-    43996 "Meetei Mayek Letter Lai Lonsum"
-    43997 "Meetei Mayek Letter Mit Lonsum"
-    43998 "Meetei Mayek Letter Pa Lonsum"
-    43999 "Meetei Mayek Letter Na Lonsum"
-    44000 "Meetei Mayek Letter Til Lonsum"
-    44001 "Meetei Mayek Letter Ngou Lonsum"
-    44002 "Meetei Mayek Letter I Lonsum"
-    44003 "Meetei Mayek Vowel Sign Onap"
-    44004 "Meetei Mayek Vowel Sign Inap"
-    44005 "Meetei Mayek Vowel Sign Anap"
-    44006 "Meetei Mayek Vowel Sign Yenap"
-    44007 "Meetei Mayek Vowel Sign Sounap"
-    44008 "Meetei Mayek Vowel Sign Unap"
-    44009 "Meetei Mayek Vowel Sign Cheinap"
-    44010 "Meetei Mayek Vowel Sign Nung"
-    44011 "Meetei Mayek Cheikhei"
-    44012 "Meetei Mayek Lum Iyek"
-    44013 "Meetei Mayek Apun Iyek"
-    44016 "Meetei Mayek Digit Zero"
-    44017 "Meetei Mayek Digit One"
-    44018 "Meetei Mayek Digit Two"
-    44019 "Meetei Mayek Digit Three"
-    44020 "Meetei Mayek Digit Four"
-    44021 "Meetei Mayek Digit Five"
-    44022 "Meetei Mayek Digit Six"
-    44023 "Meetei Mayek Digit Seven"
-    44024 "Meetei Mayek Digit Eight"
-    44025 "Meetei Mayek Digit Nine"
+    11001 "Linie înclinată dublă mai mic decât or egal cu"
+    11002 "Linie înclinată dublă mai mare decât sau egal cu"
+    11003 "Relation binară de solidus triplu"
```

Comment:
This looks only partially translated.

Reply:
44025 is already translated. It is the last translated line from this PR.